### ISSUE TITLE
Failed to convert napi value into rust type `bool`

### ISSUE THREAD

**Issue by determo13** [Atome-FE/llama-node#79]
inference.mjs

import { LLM } from "llama-node";
import { LLamaCpp } from "llama-node/dist/llm/llama-cpp.js";
import path from "path";

const model = path.resolve(process.cwd(), "../ggml-vic7b-uncensored-q5_1.bin");
const llama = new LLM(LLamaCpp);

const config = {
    path: model,
    //modelPath: model, // does not exist in llama-cpp.js instead path used 
    enableLogging: true,
    nCtx: 1024,
    seed: 0,
    f16Kv: false,
    logitsAll: false,
    vocabOnly: false,
    useMlock: false,
    embedding: false,
    useMmap: true,
    nGpuLayers: 0
};

params passed

{
  nCtx: 1024,
  seed: 0,
  f16Kv: false,
  logitsAll: false,
  vocabOnly: false,
  useMlock: false,
  embedding: false,
  useMmap: true,
  nGpuLayers: 0
}
llama-node/node_modules/llama-node/dist/llm/llama-cpp.js:65
      this.instance = yield LLama.load(path, rest, enableLogging);
                                  ^

Error: Failed to convert napi value into rust type `bool`
    at LLamaCpp.<anonymous> (file:///home/simon/llama-node/node_modules/llama-node/dist/llm/llama-cpp.js:65:35)
    at Generator.next (<anonymous>)
    at file:///home/simon/llama-node/node_modules/llama-node/dist/llm/llama-cpp.js:33:61
    at new Promise (<anonymous>)
    at __async (file:///home/simon/llama-node/node_modules/llama-node/dist/llm/llama-cpp.js:17:10)
    at LLamaCpp.load (file:///home/simon/llama-node/node_modules/llama-node/dist/llm/llama-cpp.js:61:14)
    at LLM.load (/home/simon/llama-node/node_modules/llama-node/dist/index.cjs:52:21)
    at run (file:///home/simon/llama-node/packages/llama-cpp/example-js/inference.mjs:39:17)
    at file:///home/simon/llama-node/packages/llama-cpp/example-js/inference.mjs:45:1
    at ModuleJob.run (node:internal/modules/esm/module_job:194:25) {
  code: 'BooleanExpected'
}

I guess something isn't correct with prams passed any chance to know what is the correct way to pass it?

used model below
ggml-vic7b-uncensored-q5_1.bin

**Comment by determo13** [comment]
just went to install it properly
